api/v1alpha4/zz_generated.deepcopy.go

```diff
@@ -684,6 +684,13 @@ func (in *OpenStackMachineStatus) DeepCopyInto(out *OpenStackMachineStatus) {
 		*out = new(string)
```

Comment:
After add or update files under api/v1beta1/ directory, you need to run `make generate` again.